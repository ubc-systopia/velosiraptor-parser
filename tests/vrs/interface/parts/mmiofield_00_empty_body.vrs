mmio foo [ base, 0, 8 ] {}