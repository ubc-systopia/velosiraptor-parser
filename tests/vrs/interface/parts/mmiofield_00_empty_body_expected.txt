error: empty interface field block
     --> tests/vrs/interface/parts/mmiofield_00_empty_body.vrs:1:27
      |
    1 |  mmio foo [ base, 0, 8 ] {}
      |                             remove the block or fill it in explicitly
